pkg/controller/gkenetworkparamset/gkenetworkparamset_controller.go

```diff
@@ -305,6 +314,11 @@ func (c *Controller) syncGNP(ctx context.Context, params *networkv1.GKENetworkPa
 		return nil
 	}
 
+	err = c.syncDefaultPodRanges(ctx, params)
+	if err != nil {
+		return err
+	}
```

Comment:
For readability, you can consider inlining the error handling.

Reply:
Done.